html/partials/compare/changed_config_files.html.haml

```diff
@@ -23,10 +23,14 @@
             - if @diff["changed_config_files"].only_in1
               = only_in_a
```

Comment:
I'm thinking we could keep the titles out of the `if` clause so we always show them. Otherwise having one block with a title and one without might look a bit off